code/gbq/gbq_qr.py

```diff
@@ -60,10 +64,16 @@ def main():
         # override model-specified num_bags to a smaller value
         config.num_bags = 10
         
+        # maximum forecast horizon
+        max_horizon = 3
+        
         # quantile levels at which to generate predictions
-        q_levels = [0.25, 0.50, 0.75]
-        q_labels = ['0.25', '0.5', '0.75']
+        q_levels = [0.025, 0.50, 0.975]
```

Comment:
I assume the change from .75 to .975 was intentional, but just checking

Reply:
yes, this is intentional -- makes the plots more interpretable.  thanks for checking!